androidHyperskillApp/src/main/java/org/hyperskill/app/android/home/view/ui/fragment/HomeFragment.kt

```diff
@@ -121,9 +133,20 @@ class HomeFragment :
     private fun initViewStateDelegate() {
         with(viewStateDelegate) {
             addState<HomeFeature.HomeState.Idle>()
-            addState<HomeFeature.HomeState.Loading>(viewBinding.homeScreenSkeleton.root, viewBinding.homeScreenAppBar.root)
+            addState<HomeFeature.HomeState.Loading>(
+                viewBinding.homeScreenContainer,
+                viewBinding.homeScreenSkeleton.root,
+                viewBinding.homeScreenAppBar.root
+            )
             addState<HomeFeature.HomeState.NetworkError>(viewBinding.homeScreenError.root)
-            addState<HomeFeature.HomeState.Content>(viewBinding.homeScreenContainer, viewBinding.homeScreenAppBar.root)
+            addState<HomeFeature.HomeState.Content>(
+                viewBinding.homeScreenAppBar.root,
+                viewBinding.homeScreenContainer,
+                viewBinding.homeScreenKeepPracticingTextView,
+                viewBinding.homeScreenProblemOfDayCard.root,
+                viewBinding.homeScreenTopicsRepetitionCard.root,
```

Comment:
**This is not critical**, but it would be nice if we could polish up the transition animation of the Show Learn next topic on the Home Screen (it overlaps "Continue on the web" and then jumps, I think it would be better if we will fix overlap)

[device-2023-02-03-115847.webm](https://user-images.githubusercontent.com/8586063/216516385-fa1741f9-f6db-42b3-8fd8-9afa347a42bb.webm)